script/script.js

```diff
@@ -197,7 +197,12 @@ function preserveData() {
   };
   localStorage.setItem('userInfo', JSON.stringify(userObject));
 }
-window.addEventListener('load', preserveData);
+fullName.addEventListener('focusout', preserveData);
+firstName.addEventListener('focusout', preserveData);
+lastName.addEventListener('focusout', preserveData);
+email.addEventListener('focusout', preserveData);
+fullName.addEventListener('focusout', preserveData);
+message.addEventListener('focusout', preserveData);
```

Comment:
- [ ] Good job so far 👍 but this event listener here does not seem to be working and as a result, the data cannot be preserved in a browser. Kindly fix this issue and make sure that the data is preserved in the browser so as to follow the project requirements 👍